packages/angular/projects/ui-angular/src/lib/primitives-poc/text-field/text-field-component/text-field.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="amplify-flex amplify-field amplify-textfield">
+  <!-- Label -->
+  <label [for]="inputId" *ngIf="!labelHidden">
```

Comment:
Even if `labelHidden` is `true`, `label` should still be in the DOM for accessibility purposes. It should just be visually hidden with css.

You can do so with `visuallyHidden` class: 

```html
label class="amplify-visually-hidden"
```

Reply:
This corresponds to https://ui.docs.amplify.aws/react/components/visuallyhidden element. For now, we can look to hardcode the class as `VisuallyHidden` wasn't the scope of the POC yet.

Reply:
Done.